pajbot/bot.py

```diff
@@ -785,9 +789,16 @@ def safe_say(self, message: str, channel: Optional[str] = None) -> None:
         if not self.is_bad_message(message):
             self.say(message, channel)
 
+    def safe_announcement(self, message: str, channel: Optional[str] = None) -> None:
+        if not self.is_bad_message(message):
+            self.announcement(message, channel)
+
     def me(self, message: str, channel: Optional[str] = None) -> None:
         self.say("/me " + message[: CHARACTER_LIMIT - 4], channel=channel)
 
+    def announcement(self, message: str, channel: Optional[str] = None) -> None:
+        self.say("/announce " + message[: CHARACTER_LIMIT - 4], channel=channel)
```

Comment:
The `-4` from `def me` is to account for the characters that we prefix (`/me `), so for `/announce` we'd need to cut 10 characters instead
```suggestion
        self.say("/announce " + message[: CHARACTER_LIMIT - 10], channel=channel)
```